q2_mlab/db/tests/test_db.py

```diff
@@ -37,7 +41,7 @@ def test_add(self):
         target = 'AGE'
         add(engine=engine, results=results, parameters=parameters,
             dataset=dataset, algorithm=algorithm, level=level, target=target,
-            artifact_uuid='some-uuid',
+            artifact_uuid='uuid-01',
```

Comment:
This change and the others in these tests should not have changed.

Reply:
These were breaking with current changes since the uuids were identical and weren't inserted. Can revert these changes once the uuid check is optional.